Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
Btw.: The commits of the other two open pull requests are all included in my fork.